### ISSUE TITLE
Test_CalcScaleResolution failed on aarch64

### ISSUE THREAD

**Issue by sikmir** [consbio/mbtileserver#130]
Failed on aarch64-darwin and aarch64-linux:
```
--- PASS: Test_TileCoordFromString_Invalid (0.00s)
=== RUN   Test_CalcScaleResolution
    tile_test.go:112: calcScaleResolution returned unexpected scale: 6.163099245746973e+08 expected: 6.163099245746973e+08
    tile_test.go:112: calcScaleResolution returned unexpected scale: 1.540774811436743e+08 expected: 1.540774811436743e+08
--- FAIL: Test_CalcScaleResolution (0.00s)
FAIL
FAIL    github.com/consbio/mbtileserver/handlers        0.512s
FAIL
```

**Comment by brendan-ward** [comment]
Thanks for the report!  Looks like we must be running into architecture dependent floating point comparison - though we are supposed to be only comparing to 9 decimal places, which look identical to me in the output strings above.  I think this demonstrates an error in the test case rather than the implmentation.

We need to add tests to CI for better catching these on ARM64 architectures: #131

**Comment by brendan-ward** [comment]
Fixed by d0ea8479c03d304e19ca6e4bc1b282c57d5b3108
